### PR TITLE
Add Alert docs for android

### PR DESCRIPTION
**Pull request by julbar**

### CODE REVIEW

**Review by BalbinaK — APPROVED** [review]

File: docs/components/alert/android.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+```kotlin example
+WarpAlertType {
+    Info,
+    Positive,
+    Critical,
+    Warning
+}
+```
+
+It is possible to add and optional link text, secondary button and a quiet button or all together.
```
Comment (felicia-haggqvist) [inline docs/components/alert/android.md:41]:
```suggestion
It is possible to add an optional link text, secondary button and a quiet button or all together.
```

File: docs/components/alert/android.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    linkAction = {  },
+    secondaryButtonText = "Button",
+    secondaryButtonAction = {  },
+    quietButtonText = "Quiet Button",
+    quietButtonAction = {  }
+    )
+```
+
+
+### Legacy suport
```
Comment (felicia-haggqvist) [inline docs/components/alert/android.md:58]:
```suggestion
### Legacy support
```

**Review by felicia-haggqvist — APPROVED** [review]
Just left two comments, otherwise it looks good! :)

**Comment by felicia-haggqvist** [comment]
@julbar Not sure if you had time to see my comments before you merged this PR? 🙈 Looks like the misspellings are in master :)

**Pull request merged**